### ISSUE TITLE
Oil or lotion on the skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9982]
**Preferred term label:**
Oil or lotion on the skin

**Synonyms**
Oily skin, increased oil on skin

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tinea versicolor

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
oil sounds like endogenous oil in contrast to lotion. Could you clarify the disease context?

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK482500/ "Factors that lead to this pathogenic conversion include a genetic predisposition, environmental conditions such as heat and humidity, immunodeficiency, pregnancy, oily skin, and application of oily lotions and creams."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001094
- label: Exposure to oily lotions or creams
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:29494106
- def: A history of applying oily lotions or creams to the skin.
- comment: This may be a risk factor for tinea versicolor.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9982
- synonyms:
